glove2word2vec.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Manas Ranjan Kar <manasrkar91@gmail.com>
+# Licensed under the MIT License https://opensource.org/licenses/MIT
+
+"""
+CLI USAGE: python glove2word2vec.py <GloVe vector file> <Output model file>
+
+Convert GloVe vectors into Gensim compatible format to instantiate from an existing file on disk in the word2vec C format;
+
+model = gensim.models.Word2Vec.load_word2vec_format('/tmp/vectors.txt', binary=False)  # C text format
+
+word2vec embeddings start with a line with the number of lines (tokens?) and the number of dimensions of the file. This allows gensim to allocate memory 
+accordingly for querying the model. Larger dimensions mean larger memory is held captive. Accordingly, this line has to be inserted into the GloVe 
+embeddings file.
+
+"""
+import re
+import sys
+import gensim
+import smart_open
+
+
+def glove2word2vec(glove_vector_file,output_model_file):
+
+    
+    def get_info(glove_file_name):
+        """ 
```

Comment:
Docstrings in imperative (`"Calculate X..."`, `"Return Y..."` etc). See PEP8 and PEP257.